save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/basic/ProjectStatisticMenu.kt

```diff
@@ -32,24 +38,61 @@ external interface ProjectStatisticMenuProps : Props {
      * list of tests
      */
     var latestExecutionStatisticDtos: List<TestSuiteExecutionStatisticDto>?
+
+    /**
+     * Flag to open Menu
+     */
+    var isOpen: Boolean?
 }
 
 /**
+ * @param onOpenMenuStatistic
  * @return ReactElement
  */
 @Suppress("TOO_LONG_FUNCTION", "LongMethod", "MAGIC_NUMBER")
-fun projectStatisticMenu() =
+fun projectStatisticMenu(
+    onOpenMenuStatistic: (isOpen: Boolean) -> Unit,
+) =
         fc<ProjectStatisticMenuProps> { props ->
 
+            val (latestExecutionStatisticDtos, setLatestExecutionStatisticDtos) = useState(props.latestExecutionStatisticDtos)
+            val scope = CoroutineScope(Dispatchers.Default)
+
+            if (props.isOpen == true) {
```

Comment:
I think you can add `props.isOpen` as effect's dependency and perform this check inside `useEffect`. Cleanup should occur in the same useEffect too.